### PR TITLE
Frontend: Send coins using jQuery and Ajax

### LINKED ISSUE #106: Properly send coins in faucet frontend
The "send" button in the faucet frontend is not sending coins properly, most likely due to the query parameter type.
The code has to be fixed so that the coins are properly sent to the recipient.

The following error is shown:
```
{"statusMessage":"Missing non-optional query parameter 'recv'."}
```

### PR DESCRIPTION
**Pull request by kchulj**
- If we use `jQuery` and `Ajax`, we can send coins using the correct `HTTP` method and `Content-Type`.
- The frontend now shows a validation message below the input field, depending on the given address.

1st commit pair-programmed w/ @ferencdg  
Fixes #106

### CODE REVIEW

File: frontend/index.html [hunk 1]
```diff
@@ -8,6 +8,7 @@
 
     <link rel="icon" href="favicon.ico" />
     <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-wEmeIV1mKuiNpC+IOBjI7aAzPcEZeedi5yW5f2yOq55WWLwNGmvvx4Um1vskeMj0" crossorigin="anonymous">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
```
Comment (TrustHenry) [inline frontend/index.html:11]:
```suggestion
    <script
  src="https://code.jquery.com/jquery-3.5.1.min.js"
  integrity="sha256-9/aliU8dGd2tb6OSsuzixeV4y/faTqgFtohetphbbj0="
  crossorigin="anonymous"></script>
```
Option suggestion

Using CDN maxcdn.com or Google.com or Include in Source

File: frontend/index.html [hunk 2]
```diff
@@ -19,17 +20,49 @@
...
+              <label for="recv">Enter your address</label>
             </div>
             <br>
             <div class="form-floating">
               <input name="amount" type="text" class="form-control" id="floatingInput" value="100" readonly>
               <label for="floatingInput">Amount (fixed at 100 BOA)</label>
             </div>
             <br>
-              <input class="w-90 btn btn-lg btn-primary" type="submit" value="Send test BOA to address">
+              <input id ="submitform" class="w-90 btn btn-lg btn-primary" type="submit" value="Send test BOA to address">
```
Comment (TrustHenry) [inline frontend/index.html:32]:
```suggestion
              <input id="submitform" class="w-90 btn btn-lg btn-primary" type="submit" value="Send test BOA to address">
```

File: frontend/index.html (hunk 1 above)
Comment (kchulj) [reply frontend/index.html]: Thank you for the review :)
I decided to use `code.jquery.com` CDN, as you suggested.

**Comment by kchulj** [comment]
Requesting @hewison-chris for review :)

**Review by ferencdg — APPROVED** [review]

File: frontend/index.html [hunk 3]
```diff
@@ -31,6 +31,13 @@ <h1 class="h3 mb-3 fw-normal">Welcome to the BOSAGORA Faucet</h1>
             <br>
               <input id="submitform" class="w-90 btn btn-lg btn-primary" type="submit" value="Send test BOA to address">
         </form>
+      <br>
+        <div id="valid" class="result" style="display: none;">
+          <p>Test BOA successfully sent!</p>
+        </div>
+        <div id="invalid" class="result" style="display: none;">
+          <p>Invalid address!</p>
```
Comment (hewison-chris) [inline frontend/index.html:39]: Here we are assuming the only error can be "invalid address".  What if the address is valid but it fails due to some other reason?

**Review by hewison-chris — APPROVED** [review]
LGTM, just mentioned a possible improvement (maybe for a new issue).

File: frontend/index.html (hunk 3 above)
Comment (kchulj) [reply frontend/index.html:39]: Good point!

**Pull request merged**